buildtools/webpack.commons.js

```diff
@@ -1,12 +1,13 @@
 const path = require('path');
 
-const babelPresets = [['@babel/preset-env', {
-  'targets': {
-    'browsers': ['ie 11'],
-  },
-  'modules': false,
-  'loose': true,
-}]];
+const babelPresets = [
+  ['@babel/preset-env', {
+    'targets': {
+      'browsers': ['ie 11'],
```

Comment:
Time to find a more reasonable baseline. Ex, some safari 13.
I don't think it has much impact though because people build the code with their own config.